eigrpd/eigrp_interface.c

```diff
@@ -319,7 +319,7 @@ eigrp_if_up (struct eigrp_interface *ei)
   ne->ei = ei;
   ne->reported_metric = metric;
   ne->total_metric = metric;
-  ne->distance = eigrp_calculate_metrics (eigrp, &metric);
+  ne->distance = eigrp_calculate_metrics (eigrp, metric);
```

Comment:
eigrp_metrics is 16 bytes. it's more than a 4- or 8-byte pointer, but not a real issue.